packages/core/rfcs/use-user-address-factory.md

```diff
@@ -0,0 +1,120 @@
+# UseUserAddressFactory with Commerce Tools composable use case
+
+## Motivation
+
+We need to extend useUserAddress type to show and modify user addresses.
+Also, some eCommerce platforms requires updating user entity with every address modification (for example: CommerceTools).
+
+## Composable interface
+```TS
+export interface UseUserAddress<ADDRESS, SEARCH_PARAMS, ADDRESS_OPTIONS = unknown> {
+  addresses: ComputedProperty<ADDRESS[]>;
+  shippingAddresses: ComputedProperty<ADDRESS[]>;
```

Comment:
can we have multiple shipping addressess and billing addresses? Does every address have to be in a specific category?